src/infoContributor.ts

```diff
@@ -1,22 +1,19 @@
-import * as request from 'request-promise-native'
-import { InfoContributorConfig } from './infoContributorConfig'
-
 export class InfoContributor {
-  constructor (public readonly url: string,
-               public readonly options: InfoContributorConfig = new InfoContributorConfig()) {
-  }
-
-  public call (): Promise<object> {
-    const requestConfig: request.RequestPromiseOptions | request.OptionsWithUri = { uri: this.url, json: true }
-    Object.assign(requestConfig, this.options.requestOptions)
+  constructor(public readonly url: string) {}
 
-    return request
-      .get(requestConfig)
-      .catch((err: Error) => {
-        return {
-          error: `Error calling ${this.url}`,
-          errorStackTrace: err
-        }
-      })
+  public call(): Promise<object> {
+    return fetch(this.url).then((response) => {
```

Comment:
why not use async await?

Reply:
Hmm initially I thought it wouldn't work with the router finder so I gave up trying that but thinking about it again it might work.

I'll give it another shot.

Reply:
Unfortunately the way the express.js template is setup https://github.com/hmcts/expressjs-template/blob/b5a677c573fed4d2e6f31aa4766bddf2f4511756/src/main/app.ts#L43-L46

Async routers don't work =/.

Also changing it to async is more of a breaking change and would make renovate PRs not work, with this PR it should just work.

If you have any ideas for the express.js template side I'm happy to have another look but this should be good enough

Reply:
Still waiting for that express 5 release...